skeleton_for_bot.py

```diff
@@ -0,0 +1,38 @@
+import requests
+import json
+
+TOKEN = 'YOUR_TOKEN' # insert your token from BotFather here
+BASE_URL = f"https://api.telegram.org/bot{TOKEN}/"
+LONG_POLLING_TIMEOUT = 10
+
+def bot_function():
+    ...
+
+
+def main(PLACEHOLDER_MSG="Привет") -> None:
```

Comment:
I really think that `main` function shouldn't accept any arguments apart from command line ones. And type hinting here is redundant.